src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-tutorials/opentelemetry-tutorial-java.mdx

```diff
@@ -0,0 +1,1132 @@
+
+OpenTelemetry provides an agent that automatically instruments your Java apps and doesn't require you to  modify your source code. The agent is a quick way for you to get started so you can get sample data into New Relic.
+
+The auto-instrumentation agent is a JAR file that dynamically injects bytecode to capture telemetry from popular libraries and frameworks. You can also use it to capture data such as inbound requests, outbound HTTP calls, and database calls. It can be attached to any Java 8+ application.
+
+<Callout variant="tip">
+  See the [official documentation](https://opentelemetry.io/docs/instrumentation/java/automatic/) for additional configuration options.
+</Callout>
+
+To monitor our demo app with the OpenTelemetry Java agent:
+
+1. Download the [agent JAR file](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/latest/download/opentelemetry-javaagent.jar), which contains the agent and instrumentation libraries. Place it in your preferred directory and note the path, which you will use later.
+2. If you haven't already done so, complete the following:
+    ```
+    git clone git@github.com:newrelic/newrelic-opentelemetry-examples.git
+    cd newrelic-opentelemetry-examples/getting-started-guides/java/Uninstrumented
+    ```
+3. Build the demo app with this command:
+    ```
+    ./gradlew build
+    ```
+    <Callout variant="tip">
+      After a successful build, you will find the resulting application JAR file in `Uninstrumented/build/libs`.
+    </Callout>
+
+4. Export the following environment variables to configure the agent. Click on each for details:
+
+    <CollapserGroup>
+      <Collapser
+        className="freq-link"
+        id="exp-bucket-hist"
+        title="OTEL_EXPORTER_OTLP_HEADERS=api-key=YOUR_LICENSE_KEY"
+      >
+        Headers: This determines which New Relic account your data will be sent to. Replace `YOUR_LICENSE_KEY` with your [ingest license key](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/#license-key).
+      </Collapser>
+      <Collapser
+        className="freq-link"
+        id="temp"
+        title="OTEL_EXPORTER_OTLP_ENDPOINT=https://otlp.nr-data.net:4317 (For the US)"
+      >
+        This is the endpoint your application telemetry will be exported to. Export data to New Relic's OTLP endpoint. If you're in the EU, use this endpoint:
+
+        OTEL_EXPORTER_OTLP_ENDPOINT=https://otlp.eu01.nr-data.net:4317
+      </Collapser>
+      <Collapser
+        className="freq-link"
+        id="service-name"
+        title="OTEL_SERVICE_NAME=getting-started-java"
+      >
+        This is the name that associates your application telemetry with an entity in New Relic. In this tutorial we're calling it `getting-started-java`, but you could call it anything you want.
+      </Collapser>
+      <Collapser
+        className="freq-link"
+        id="instance-id"
+        title="OTEL_RESOURCE_ATTRIBUTES=service.instance.id=INSERT_YOUR_OWN_ID_HERE"
```

Comment:
think you could remove 'INSERT' as we're not using elsewhere. austin said he thought it was overkill in general

Reply:
I'm gonna leave this since I don't think it's too verbose. By having a verb in the variable, I'm hoping it catches the reader's attention.